inc/zoo/swar/SWAR.h

```diff
@@ -68,7 +76,21 @@ struct SWAR {
                 ~(~T(0) << NBits),
         // Use LowerBits in favor of ~MostSignificantBit to not pollute
         // "don't care" bits when non-power-of-two bit lane sizes are supported
-        LowerBits = MostSignificantBit - LeastSignificantBit;
+        LowerBits = MostSignificantBit - LeastSignificantBit,
+        MaxUnsignedLaneValue = LeastSignificantLaneMask;
+
+    template <typename U>
+    constexpr static auto from_array(const U (&values)[Lanes]) noexcept {
+        auto result = T{0};
+        for (auto value : values) {
+            result = (result << NBits) | value;
```

Comment:
blitElement?

Reply:
I this does not quite blit the element, if it will move in an element, it shifts up what it got and then adds to the least significant lane